src/components/ProofOfAddress/PoAIntro/index.js

```diff
@@ -8,37 +8,51 @@ import { POA_INTRO_LOCALES_MAPPING } from '~utils/localesMapping'
 import theme from '../../Theme/style.scss'
 import style from './style.scss'
 
-const PoAIntro = ({ country, translate, parseTranslatedTags, nextStep }) => (
-  <div className={theme.fullHeightContainer}>
-    <PageTitle
-      title={translate('poa_intro.title', {
-        country: !country || country === 'GBR' ? 'UK' : '',
-      })}
-    />
-    <div className={style.content}>
-      <p className={style.requirements}>{translate('poa_intro.subtitle')}</p>
-      {['shows_address', 'matches_signup', 'most_recent'].map((key) => (
-        <div key={key} className={style.requirement}>
-          <span>
-            {parseTranslatedTags(POA_INTRO_LOCALES_MAPPING[key], ({ text }) => (
-              <span className={style.bolder}>{text}</span>
-            ))}
-          </span>
-        </div>
-      ))}
+const PoAIntro = ({
+  country,
+  translate,
+  parseTranslatedTags,
+  nextStep,
+  steps,
+  autoFocusOnInitialScreenTitle,
+}) => {
+  const isFirstScreen = steps[0].type === 'face'
```

Comment:
Why do we assume that face is the first screen on the PoA flow?

Reply:
Ah, good spot. I meant for it to be `'poa'`, not `'face'` in this case.